data/json/npcs/classes.json

```diff
@@ -131,6 +131,32 @@
       { "skill": "rifle", "bonus": { "rng": [ 0, 2 ] } }
     ]
   },
+  {
+    "type": "npc_class",
+    "id": "NC_MARLOSS_VOICE",
+    "name": "Marloss Voice",
+    "job_description": "I spread the Hymns so that peace and unity return to our world.",
+    "//": "THRESH_MYCUS is here only for faction AI purposes. This NPCS is not meant to be part of the fungaloid mind, they have just eaten marloss berries and maybe seeds or gelatin ",
```

Comment:
```suggestion
    "//": "THRESH_MYCUS is here only for faction AI purposes. This NPC is not meant to be part of the fungaloid mind, they have just eaten marloss berries and maybe seeds or gelatin ",
```

Reply:
Removed the comment because the explanation is no longer necessary